### ISSUE TITLE
Consider case-insensitive `name` and `description` properties

### ISSUE THREAD

**Issue by stevage** [umap-project/umap#223]
I imported some GeoJSON files which, for whatever reason, have `Name` and `Description` (instead of `name` and `description`).

![screenshot 2015-10-11 19 03 35](https://cloud.githubusercontent.com/assets/678344/10415920/e6fb1160-704a-11e5-8a6d-2e43cba3e5da.png)

umap basically ignores them. Perhaps it should quietly convert `Name` to `name` if there is no `name` field?

**Comment by yohanboniface** [comment]
> umap basically ignores them. Perhaps it should quietly convert Name to name if there is no name field?

Yeah, that something in the air, also for mapping from other languages ("nom" in French).
On thing I'd like to do also is to be able to rename a field for all feature of a layer.

**Comment by stevage** [comment]
Ok, cool. Although those things sound quite a bit more complex. But "more correct".

**Comment by yohanboniface** [comment]
Actually, have you tried renaming with the table editor?

**Comment by stevage** [comment]
Oh, that works! To be honest, I didn't know there was a table editor until an hour ago when I went back and read some of your blog posts :)

Maybe making the property names directly editable from the layer properties page, or having a link to launch the table editor would make this workaround more obvious?

**Comment by yohanboniface** [comment]
> Maybe making the property names directly editable from the layer properties page

I'll consider that.

> or having a link to launch the table editor would make this workaround more obvious?

Yes, that's missing. :)

**Comment by wpkurtz** [comment]
This thread just saved me so much time! Thank you so much for pointing out the case sensitivity!
